phileas-model/src/main/java/ai/philterd/phileas/model/policy/filters/strategies/AbstractFilterStrategy.java

```diff
@@ -107,6 +110,18 @@ public abstract class AbstractFilterStrategy {
     @Expose
     protected String maskLength = SAME;
 
+    @SerializedName("truncateDigits")
+    @Expose
+    protected Integer truncateDigits;
```

Comment:
Do you think `truncateLength` is more descriptive? I noticed on line 393 you did refer to it as length in the error message.

When I first read it I interpreted it as "the specific digits you want to truncate" even though that doesn't make a lot of sense.

Reply:
Oh, yes, `truncateDigits` is what's in the zip code filter. I forgot about that. That used to make sense and now I don't like it.

I see your point. `truncateLeaveCharacters` makes sense. I think that's clearer. I'll write a new issue to make the property in `ZipCodeFilterStrategy` consistent with what you use here.